### PR TITLE
Show Add-ons section at all times.

### PR DESCRIPTION
**Pull request by reddyashish**
This PR is to address https://jira.autodesk.com/browse/DYN-2327. 

The Add-ons section header will be shown at all times. Users can import new libraries even when there are add-ons already present.

Screen Shot 2022-02-02 at 3 46 16 AM

Screen Shot 2022-02-02 at 3 55 52 AM

@mjkkirschner @aparajit-pratap

### CODE REVIEW

File: src/LibraryUtilities.ts [hunk 1]
```diff
@@ -562,11 +562,11 @@ export function removeEmptyNodes(items: ItemData[]) {
             if (removeEmptyNodes(item.childItems)) {
                 i--;
             }
-        } else if (item.itemType === "section" || item.itemType === "category" || item.itemType === "group" || item.itemType === "coregroup") {
+        }else if ( (item.itemType === "section" && item.contextData !== "Add-ons") || item.itemType === "category" || item.itemType === "group" || item.itemType === "coregroup") {
```
Comment (mjkkirschner) [inline src/LibraryUtilities.ts:565]: can you add a comment to the summary of the method that it has special rules for the "Add-ons" section...

**Comment by mjkkirschner** [comment]
@reddyashish please add a test of this if possible, you may also need to update the snapshots for the existing tests.

**Comment by reddyashish** [comment]
@mjkkirschner Modified one of the tests to add this case. All the librarie.js tests are passing.

**Comment by mjkkirschner** [comment]
@reddyashish snapshot tests also pass?

**Review by mjkkirschner — APPROVED** [review]

**Pull request merged**